index.html

```diff
@@ -341,8 +345,48 @@ <h2>Skills</h2>
           <hr>
         </div>
       </section>
-      <div class="workDivider"></div>
+
       <!-- contact us section -->
+      <section class="contact">
+        <div class="contact-container">
+          <h2>Contact me</h2>
+
+          <p class="contact-P">If you have an application you <br>are interested in developing, a feature that you
+            need to built or a
+            project that needs coding. i'd love to help with it</p>
+
+          <!-- form section -->
+
+          <form action="" method="post" class="contact-form" action="https://formspree.io/f/xknagbbo">
```

Comment:
- [ Required ] It would be better if your contact form has only one `action` attribute, currently it has two. The empty action attribute prevents form submission to [https://formspree.io/f/xknagbbo](https://formspree.io/f/xknagbbo). Please make sure you have one `action` attribute on the form and it has the value [https://formspree.io/f/xknagbbo](https://formspree.io/f/xknagbbo)
```HTML
  <form action="https://formspree.io/f/xknagbbo" method="post" class="contact-form">
```

Reply:
oh thank you for bringing it to my notice
@Felix45